scripts/ipfs/uninstall

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Uninstall IPFS
+sudo systemctl disable ipfs-daemon.service
+sudo systemctl stop ipfs-daemon.service 2>/dev/null || true
+sudo rm -f /usr/local/bin/ipfs
+sudo rm -f /lib/systemd/system/ipfs-daemon.service
+
+if [ -d "/home/pi/.ipfs" ]; then
+    echo "Found /home/pi/.ipfs"
+    read -p "Remove your IPFS data (Y/n)? " RM_IPFS_HOME;
```

Comment:
@garrying Change this to: **Keep your IPFS data (Y/n)?**
`<empty>`, `Y`, `y`, `Yes`, `yes` all leave the file untouched, otherwise remove?